lib/src/base-styles/_nx-tables.scss

```diff
@@ -10,141 +10,145 @@
+        border-bottom-style: none;
       }
 
-      .nx-cell--label, .nx-cell--chevron {
-        color: $nx-text-color;
+      > :first-child {
+        border-bottom-left-radius: $nx-border-radius;
       }
 
+      > :last-child {
+        border-bottom-right-radius: $nx-border-radius;
+      }
     }
 
-    &:first-child .nx-cell {
-      border-top-style: none;
-    }
+    &.nx-clickable {
+      &:hover {
+        > .nx-cell {
+          background-color: #f1f3fa;
+        }
+      }
+
+      &.selected, &:active {
+        > .nx-cell {
+          background-color: #d3e6ff;
+        }
+      }
 
-    .nx-cell--label {
-      @include semi-bold();
+      &:focus {
+        box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+        outline: none;
 
-      color: $nx-text-color;
+        > .nx-cell {
+          border-color: #02a6d9;
+        }
+      }
     }
   }
 }
 
-.nx-table-row--header {
-  border-bottom: $nx-border-row-dark;
+.nx-table--scrollable {
+  border-style: none;
+  border-radius: 0;
+  overflow: visible;
+
+  > thead > :first-child, > tbody > :last-child, {
+    > .nx-cell {
+      &:first-child, &:last-child {
+        border-radius: 0;
+      }
+    }
+  }
+
+  // create a stacking context so the headers don't appear over other content outside the table
+  position: relative;
+  z-index: 0;
+
+  .nx-cell--header {
+    position: sticky;
+    top: 0;
+    z-index: 1;
+  }
 }
 
 .nx-cell {
   @include container-vertical;
   @include container-horizontal;
 
-  background-clip: padding-box;
-  border-top: $nx-border-row;
-  box-sizing: border-box;
-  color: $nx-text-color;
-  display: table-cell;
-  height: 39px; // 38px cell, 1px top border
-  padding: $nx-spacing-xs $nx-spacing-sm;
-  position: relative;
-  text-align: left;
+  padding: $nx-spacing-sm $nx-spacing-md;
+}
 
-  &.nx-error {
-    border-color: $nx-fail-color-bg;
-    border-bottom-color: $nx-fail-color;
-    color: $nx-text-color-error;
-    text-align: center;
-  }
+.nx-cell--label {
+  @include semi-bold();
 }
 
 .nx-cell--header {
-  @include semi-bold;
-  border-bottom: $nx-border-row-dark;
-  background-color: $nx-off-white;
+  @include bold;
+  background-color: #bbc1de;
   border-top: none;
+  box-sizing: border-box;
 
-  color: $nx-text-color;
-  font-size: 12px;
-  height: 24px;
-  line-height: 12px;
+  color: $nx-text-color-dark;
+  font-size: $nx-font-size-s;
+  height: 48px;
   text-align: left;
   text-transform: uppercase;
-}
-
-.nx-cell--label {
-  @include semi-bold();
-
-  color: $nx-text-color;
+  white-space: nowrap;
 }
 
 // columns whose content is numerical
 .nx-cell--num {
   text-align: center;
 }
 
-// Used when table contents are empty
-.nx-cell--empty {
-  @include regular();
-
-  color: $nx-text-color;
+// Used for table cells containiner meta-information such as the fact that the table is empty, or
+// that the data is loading. These cells are expected to take up the entire row using colspan attributes
+.nx-cell--meta-info {
```

Comment:
Zeplin shows the error, empty, and alert cells as having 16px of top spacing

Reply:
> Zeplin shows the error, empty, and alert cells as having 16px of top spacing

78c2c7f0